jf_agent/jf_jira/jira_download.py

```diff
@@ -684,10 +684,8 @@ def download_customfieldoptions(jira_connection, project_ids):
             meta = jira_connection.createmeta(
                 projectIds=[project_id], expand='projects.issuetypes.fields'
             )
-        except JIRAError:
-            agent_logging.log_and_print_error_or_warning(
-                logger, logging.ERROR, error_code=3072, exc_info=True
-            )
+        except JIRAError as e:
+            logger.warning(f"{e}. Error calling createmeta JIRA endpoint. Skipping...")
```

Comment:
Better to continue to use `agent_logging.log_and_print_error_or_warning` (as per advice at the top of `agent_logging.py`), but just switch the level from `logging.ERROR` to `logging.WARNING`.

Reply:
And also remove the `exc_info=True` -- better to not show the stack trace